.gitattributes

```diff
@@ -0,0 +1,56 @@
+I# Set the default behavior, in case people don't have core.autocrlf set.
```

Comment:
"I" before # is a typo, similar to https://github.com/SpineEventEngine/todo-list/pull/35